Invalid read of size 4
 at oob_read_test
 by main
 Address is 0 bytes after a block of size 40 alloc'd
 at malloc
 by oob_read_test
 by main
Invalid read of size 4
 at oob_read_test
 by main
 Address is 4 bytes before a block of size 40 alloc'd
 at malloc
 by oob_read_test
 by main
Invalid write of size 4
 at oob_write_test
 by main
 Address is 0 bytes after a block of size 40 alloc'd
 at malloc
 by oob_write_test
 by main
 Conditional jump or move depends on uninitialised value(s)
 at umr_heap_test
 by main
 Conditional jump or move depends on uninitialised value(s)
 at umr_stack_test
 by main
Invalid read of size 4
 at use_after_free_test
 by main
 Address is 20 bytes inside a block of size 40 free'd
 at free
 by use_after_free_test
 by main
Invalid write of size 4
 at use_after_free_with_reuse_test
 by main
 Address is 24 bytes inside a block of size 40 free'd
 at free
 by use_after_free_with_reuse_test
 by main
VG_USERREQ__CLIENT_CALL1: func=0x0
VG_USERREQ__CLIENT_CALL2: func=0x0
VG_USERREQ__CLIENT_CALL3: func=0x0
 BACKTRACE:
 at VALGRIND_PRINTF_BACKTRACE
 by test_backtrace
 by test_backtrace1
 by test_backtrace2
 by test_backtrace3
 by main
VG_USERREQ__CLIENT_CALL1: func=0x0
 many_args BACKTRACE:
 at VALGRIND_PRINTF_BACKTRACE
 by zz_wrap_me
 by zz_wrap_me
 by zz_wrap_me0
 by many_args_wrapping_test
 by main
strcmp: 0
strcmp: -1
strcmp: 1
ERROR SUMMARY: 7 errors from 7 contexts
